src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/JurisdictionServiceImpl.java

```diff
@@ -54,11 +57,18 @@ public JurisdictionUserCreationRequest createJurisdictionUserProfileRequestForSu
     public void callCcd(JurisdictionUserCreationRequest request, String userId) {
         String s2sToken = authTokenGenerator.generate();
         try (Response response = jurisdictionFeignClient.createJurisdictionUserProfile(userId, s2sToken, request)) {
-            log.info("Jurisdiction create user profile success!!");
+            if (response == null || response.status() > 300) {
+                log.info("CCD status code: " + (response == null ? null : response.status()));
+                throwException(null);
+            }
```

Comment:
we are not passing correct info here (what would we print  when 403 scenario..internal server or forbidden)
when response null pass 500 status code directly instead of null

Reply:
Instead of null printing 500, other logic seems correct.

Reply:
fixed